src/services/RuneLiteService.ts

```diff
@@ -83,6 +87,7 @@ class RuneLiteService {
         ...newPlayerData.value.skills,
         ...data.data.level,
       },
+      collectionLog: base64Union(newPlayerData.value.collectionLog ?? "", data.data.collectionLog ?? "")
```

Comment:
I don't think this is what we want to do long term since this means if we ever make a mistake in the manifest order it will not be fixable without intervention on our part. Probably what we really want here is
```
!!newPlayerData.value.collectionLog ? newPlayerData.value.collectionLog : (data.data.collectionLog ?? "")
```
but I've already tested it as is, and this doesn't have longterm effects if we get it slightly wrong on launch. This made sense when we were thinking we might do partial syncs if we saw an ingame message, but we have punted on that for now.